client/src/App.js

```diff
@@ -1,30 +1,28 @@
-import React, { Component } from 'react';
 import './App.css';
-import Project from './Project';
+import React, {Component} from 'react';
 import Header from './Header';
+import Project from './Project';
 
 class App extends Component {
   constructor(props) {
     super(props);
-    this.state = {
-      projects: [],
-    };
+    this.state = {projects: []};
     this.submitNewProject = this.submitNewProject.bind(this);
     this.showProjects = this.showProjects.bind(this);
     this.toggleForm = this.toggleForm.bind(this);
   }
 
-  componentWillMount() {
+  UNSAFE_componentWillMount() {
```

Comment:
Whoa. Does this still get called if it's got a different name?

Reply:
it's the same function, they deprecated the other - trying to make us not use it